Resources/Private/BackendModule/src/Util/defaultModuleConfiguration.ts

```diff
@@ -0,0 +1,15 @@
+export default {
```

Comment:
Können wir da nicht nur die Konfiguration aus dem Backend nutzen?
Ich versuche zu vermeiden, dass man an so vielen komplett unterschiedlichen Stellen im Code etwas ändern muss, um ein neues Modul hinzuzufügen.

Reply:
Schlägt in dieselbe Kerbe was wir gestern am Telefon besprochen haben, wonach wir in den Settings.yaml eine Default-Konfiguration und eine Custom-Konfiguration und dann im Modul selbst eine User-Konfiguration haben. Dann wäre alles relevante in YAML.